tests/ui/js/base-page-storyEditTests.js

```diff
@@ -408,22 +424,23 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                     "args": "onAllUiComponentsReady event fired."
                 },
                 {
-                    func: "sjrk.storyTelling.testUtils.verifyPageVisibility",
+                    func: "sjrk.storyTelling.testUtils.verifyStepVisibility",
                     args: [
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage2", "{storyEdit}.storyPreviewer.container"],
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage1"]
+                        ["{storyEdit}.storyEditor.dom.storyMetadataStep", "{storyEdit}.storyPreviewer.container"],
+                        ["{storyEdit}.storyEditor.dom.storyEditStoryStep"]
                     ]
                 },
                 {
-                    "jQueryTrigger": "click",
-                    "element": "{storyEdit}.storyEditor.dom.storyEditorNext"
+                    jQueryTrigger: "click",
+                    element: "{storyEdit}.storyEditor.dom.storyEditorNext"
                 },
                 {
-                    "event": "{storyEdit}.storyEditor.events.onVisibilityChanged",
-                    "listener": "sjrk.storyTelling.testUtils.verifyPageVisibility",
-                    "args": [
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage1", "{storyEdit}.storyPreviewer.container"],
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage2"]
+                    changeEvent: "{storyEdit}.storyEditor.applier.modelChanged",
```

Comment:
Add a visibility test for clicking the "previous" button from the metadata step to the edit story step.